distribution/scenes/textures/patterns/quilt2.pov

```diff
@@ -2,125 +2,130 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Quilted pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Quilted pattern example quilt2.pov.
+//
 // -w800 -h600 +a0.3
 
 #version 3.7;
-
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
+#declare White = srgbft <1.00,1.00,1.00,0,0>;
```

Comment:
It is my habit to code the full 5, but the above is a mistake. I meant to use srgb - or the shortest srgb* form for each situation. I'll push an update for this one and others I find.